sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -77,6 +77,33 @@ char *ottf_task_get_self_name(void) {
   return pcTaskGetName(/*xTaskToQuery=*/NULL);
 }
 
+// TODO where should this go?
```

Comment:
perhaps this could be a parameter in the OTTF test config [struct](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_framework/ottf_test_config.h;drc=ac7441d809e931f14f33f61aa5dcbbd755d1b2d8;l=81), and you can have the default of 10 be set in the OTTF_DEFINE_TEST_CONFIG [macro](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_framework/ottf_test_config.h;drc=ac7441d809e931f14f33f61aa5dcbbd755d1b2d8;l=99)?